src/components/Landing/QAs.tsx

```diff
@@ -65,15 +133,21 @@ export default function QAs() {
     {
       q: t`Can I change my project's contract after it's been created?`,
       a: [
-        t`A project owner can propose changes to any part of the contract at any time, with changes taking effect after the current budgeting time frame has ended.`,
-        t`For now, a minimum of 14 days must pass from the time of a proposed reconfiguration for it to take effect. This gives token holders time to react to the decision.`,
+        t`A project owner can propose changes to any part of the contract at any time, with changes taking effect in the project's next funding cycle.`,
+        t`The owner can also configure how long these changes must be made before the next funding cycle in order for them to take effect. This gives token holders time to react to the decision.`,
```

Comment:
Not perfect, but wdyt about:

```suggestion
        t`The owner can also configure a relative due date for when configurations must be submitted before the next funding cycle. This gives token holders time to react to the decision.`,
```

Reply:
How about:

"However, to assure their community against rug-pulls, the owner can also configure a minimum time period before the next funding cycle for which reconfigurations must be made in order for them to be able to take effect. In other words, this prevents the owner from changing the funding configuration right before the next funding cycle, and not giving their token holders a chance to react by proposing further changes or redeeming their tokens."

Also, I'm wondering if we need that last sentence:

"Anyone can deploy and use another governance smart contract to override this scheme." 

WDYT?

Reply:
@johnnyd-eth Haha, I'm finding this copy a challenge! What about:

"Owners can configure a delay period, meaning reconfigurations to an upcoming funding cycle must be submitted a certain number of days before it starts. This gives token holders time to react to the decision and reduces the chance of rug-pulls."

Or we could also include an example:

"Owners can configure a delay period, meaning reconfigurations to an upcoming funding cycle must be submitted a certain number of days before it starts. For example, a 3-day delay period means reconfigurations must be submitted at least 3 days before the next funding cycle starts. This gives token holders time to react to the decision and reduces the chance of rug-pulls."

cc @peripheralist @JayZeeDesign

Reply:
I like it but would prefer last sentence to be something like "This gives token holders time to react to the decision, acting as a safeguard against rug-pulls."

Reply:
All these suggestions seem solid. Personally I like including examples where we can, and keeping things as brief as possible. I think this is my favorite:

> "Owners can configure a delay period, meaning reconfigurations to an upcoming funding cycle must be submitted a certain number of days before it starts. For example, a 3-day delay period means reconfigurations must be submitted at least 3 days before the next funding cycle starts. This gives token holders time to react to the decision and reduces the chance of rug-pulls."

Reply:
> acting as a safeguard against rug-pulls.

@johnnyd-eth the thing is, a 3-delay _could_ be a "slow" rug-pull, it's not a guaranteed safe-guard